packages/jaeger-ui/src/components/SearchTracePage/index.js

```diff
@@ -44,6 +43,8 @@ const TabPane = Tabs.TabPane;
 
 // export for tests
 export class SearchTracePageImpl extends Component {
+  pollingTimer = null;
```

Comment:
name is too general, this component deals with other things as well

Reply:
I'll change it